castervoice/apps/gitbash.py

```diff
@@ -121,9 +129,17 @@ class GitBashRule(MergeRule):
               rdescript="GREP: Search Recursive Filetype"),
         "to file":
             R(Text(" > FILENAME"), rdescript="Bash: To File"),
+        
+        # Folder path commands (not git specific)
+        "[folder] path <folder_path>": 
+            R(Text("%(folder_path)s"), rdescript="GIT: type in folder path"),
+        "(CD | go to) <folder_path>": 
+            R(Text("cd %(folder_path)s") + Key("enter"), rdescript="GIT: go to folder"),
+            
```

Comment:
I quite like having "bring me" and "navigate to" as options here too

Reply:
If I call it "bring me" ( as one of the alternatives alongside "CD", "navigate to", "go to" ) it will typically override the usual bring me command which opens up a window in Windows Explorer. That sounds fine to me. Another option would be to make it something like "shell bring me".

Reply:
Once we get bring me to search for open instances and work with terminals we might have to revisit this. I'm happy to leave it in for now.